app/uk/gov/hmrc/servicedependencies/persistence/BobbyRulesSummaryRepository.scala

```diff
@@ -60,11 +60,21 @@ class BobbyRulesSummaryRepository @Inject()(
       .map(_.headOption)
       .flatMap {
         case Some(a) => Future(Some(a))
-        case None    => getHistoric.map(_.headOption)
+        case None    => getMostRecent()
+      }
+
+  def getMostRecent() : Future[Option[BobbyRulesSummary]] =
+    collection.find()
+      .sort(descending("date"))
+      .first()
+      .toFuture()
+      .map(Option.apply)
+      .recover {
+        case _ => None
       }
 
```

Comment:
in this case the fallback was bringing all the records back and selecting the most recent which as the db grew was a lot of data that needed to be sent/serialzied etc